src/scala/org/pantsbuild/zinc/compiler/BUILD

```diff
@@ -1,6 +1,9 @@
 # Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
+# NB: To build a native-image of zinc *for a specific scalac version*, the scala path
```

Comment:
Currently these can be specified at runtime; does this mean we need to stop accepting these flags (both in pants and in our zinc wrapper)?

Reply:
If you passed the flags to pants, you would presumably want a new image bootstrapped.

But yea, would need to remove the flags from zinc.

Reply:
What #6893 does is generate native images on demand, and I almost but didn't quite get to just making that into its own executor yet. I think the approach of making a separate execution strategy for this will work and can be done at runtime. This is still an open question right now.